src/shogun/preprocessor/RandomFourierGaussPreproc.h

```diff
@@ -22,8 +22,7 @@ namespace shogun {
  *
  * approximation quality depends on dimension of feature space, NOT on number of data.
  *
- * effectively it requires two parameters for initialization: (A) the dimension of the input features stored in
- * dim_input_space
+ * effectively it requires only two parameters for initialization: (A) the width of the kernel
  * (B) the dimension of the output feature space
  *
  * in order to make it work there are two ways:
```

Comment:
I'd like your view on this, especially the case 1.

Reply:
note sure what you mean, but yeah the input dim is clear from passed data in fit

Reply:
```
 * (1) if you have already previously computed random fourier features which you want to use together with
 * newly computed ones, then you have to take the random coefficients from the previous computation and provide them
 * via void set_randomcoefficients(...) for the new computation
 * this case is important for example if you compute separately features on training and testing data in two featur
```
This thing, somehow there has been special emphasis on passing on the random coefficients. I didn't quite get it.

Reply:
just remove. This use case doesnt exist .... ;)